### PR TITLE
Evo 676 reacts on comments

### LINKED ISSUE #676: Change upvoting to emoji reactions
Do we want a limited set like FB or all of them like Slack?

### PR DESCRIPTION
**Pull request by thomasgwatson**
closes #676

### CODE REVIEW

File: src/components/PostListRow/PostListRow.js [hunk 1]
```diff
@@ -43,10 +42,10 @@ const PostListRow = (props) => {
 
   return (
     <div styleName={cx('post-row', { unread, expanded })} onClick={showDetails}>
-      <div styleName='votes'>
+      {/* <div styleName='votes'>
         <a
-          onClick={(e) => {
-            voteOnPost(e)
+          onClick={(e) => { // TODO: all of this needs to be tweaked
```
Comment (thomasgwatson) [inline src/components/PostListRow/PostListRow.js:47]: Haven't gotten to PostListRow or Grids views. Do we want to try tackling them now as well???

File: src/routes/PostDetail/PostDetail.connector.js [hunk 2]
```diff
@@ -49,22 +48,22 @@ export function mapDispatchToProps (dispatch, props) {
...
     leaveProject: () => dispatch(leaveProject(postId)),
-    voteOnPost: (myVote) => dispatch(voteOnPost(postId, myVote)),
     processStripeToken: (postId, token, amount) => dispatch(processStripeToken(postId, token, amount)),
     respondToEvent: response => dispatch(respondToEvent(postId, response))
   }
 }
 
-export function mergeProps (stateProps, dispatchProps, ownProps) {
-  const { post } = stateProps
+// export function mergeProps (stateProps, dispatchProps, ownProps) {
```
Comment (thomasgwatson) [inline src/routes/PostDetail/PostDetail.connector.js:56]: I don't think any of this is needed but I'm not completely sure how to test. 

There are no props being merged in the 'mergeProps' function, sooo.... delete?

File: src/routes/PostDetail/PostDetail.js [hunk 3]
```diff
@@ -158,103 +158,112 @@ export default class PostDetail extends Component {
...
-            <div styleName='project-management-tool'>
-              <div>View project management tool</div>
-              <div><a styleName='project-button' href={post.projectManagementLink} target='_blank'>View tasks</a></div>
-            </div>
-          }
-          {post.donationsLink && donationService &&
-            <div styleName='donate'>
-              <div>Support this project on <img src={`/assets/payment-services/${donationService}.svg`} /></div>
-              <div><a styleName='project-button' href={post.donationsLink} target='_blank'>Contribute</a></div>
+    return (
```
Comment (thomasgwatson) [inline src/routes/PostDetail/PostDetail.js:168]: I think this change is all linting

**Review by tibetsprague — APPROVED** [review]
looks good ! but lets hold off on merging. still waiting on grid items too?

**Comment by lorenjohnson** [comment]
Hey @thomasgwatson -- just catching-up to this. Excited for it to move forward and this all looks good!

Once you we get closer to merging please hmu re. the ORM Reducer merge conflicts and general App considerations. I don't expect they're anything hairy, but I'll be copying at least the reducer changes into the App. 

If we don't find the moment before it's merged that's fine too, but it'd be best to catch it then.

**Comment by tibetsprague** [comment]
already merged with chat

**Pull request closed without merging**